changelog/738.feature.rst

```diff
@@ -0,0 +1,4 @@
+Add aliases to the ``original_message`` methods.
```

Comment:
I don't know, but maybe you want to change it to this
```suggestion
Add aliases to the ``*_original_message`` methods.
```

Reply:
I intentionally didn't do that since it looks odd for this use case, and one of the methods is named `original_message`

Reply:
Ah, good point, though you are technically naming both `original_message` and other functions that go other that name 🤷🏼‍♂️

Reply:
Yes. However, the goal here is only to shift to the new functions, and then deprecate the old ones in the next release.

Reply:
👍🏼